src/booleans.js

```diff
@@ -1,62 +1,83 @@
-function negate(a) {
-  // your code here
-};
+git function negate(a) {
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  return a && b;
+}
 
 function either(a, b) {
-  // your code here
-};
+  return a || b;
+}
 
 function none(a, b) {
-  // your code here
-};
+  return !(a || b);
+}
 
 function one(a, b) {
-  // your code here
-};
+  return (a && !b) || (!a && b);
+}
 
 function truthiness(a) {
-  // your code here
-};
+  if (a) {
+    return true;
+  } else {
+    return false;
+  }
+}
 
 function isEqual(a, b) {
-  // your code here
-};
+  return a === b;
+}
 
 function isGreaterThan(a, b) {
-  // your code here
-};
+  return a > b;
+}
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
-};
+  return a <= b;
+}
 
 function isOdd(a) {
-  // your code here
-};
+  return Math.abs(a % 2) == 1;
+}
 
 function isEven(a) {
-  // your code here
-};
+  return a % 2 == 0;
+}
 
 function isSquare(a) {
-  // your code here
-};
+  return Number.isInteger(Math.sqrt(a));
+}
 
 function startsWith(char, string) {
-  // your code here
-};
+  return string.charAt(0) == char;
+}
 
 function containsVowels(string) {
-  // your code here
-};
+  let vowelCount = 0;
+  const vowels = ['a', 'e', 'i', 'o', 'u'];
+  const lowString = string.toLowerCase();
+
+  for (let char of lowString) {
+    if (vowels.includes(char)) {
+      vowelCount++;
+    }
+  }
+  if (vowelCount == 0) {
+    return false;
+  } else {
+    return true;
+  }
+}
 
 function isLowerCase(string) {
-  // your code here
-};
+  if (string.toLowerCase() !== string) {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
instead of putting this inside of an `if` statement, you could just return the condition, e.g.
```
return string.toLowerCase() === string;
```